taui/src/components/form.js

```diff
@@ -179,7 +196,8 @@ class Form extends React.PureComponent<Props> {
             placeholder={t('Map.SelectNetwork')}
             style={SELECT_STYLE}
             wrapperStyle={SELECT_WRAPPER_STYLE}
-            value={network}
+            // convert network name to match react-i18next's reference phrase and translate
+            value={{...network, label: t('Map.NetworkOptions.' + network.networkName.split(' ').join(''))}}
```

Comment:
Since this also uses `react-virtualized-select` I _think_ it should be possible to just use `network.url` here.

Reply:
While setting the Select menu `value` to `destination.value` did work for the 'from' menu, setting the Select value to `network.url` here only partially did in that it does allow you to update your network but it does not display a default value or the selected value, using the 'Set time of day' placeholder message instead. Despite trying for quite a while to figure out the difference between the two, I'm still not entirely sure why; my best guess is that it comes down to the difference between the location menu's `updateOrigin` vs the network menu's `setActiveNetwork` functions where `updateOrigin` is being given the entire `destination` object and `setActiveNetwork` is only being given a string. My guess is that those functions eventually pass these values back to the menus to display both the default value on page load and following a selection and if state only has a piece of the object, as in this menu, it displays the placeholder option instead.

That being said, what's the best way to move forward with this? I can keep exploring to figure out exactly what's happening or attempt to get `setActiveNetwork` to take in more than a string; what do you think are next steps with this?

Reply:
Darn, that's too bad. I don't think we need to spend a ton of effort on this particular point; it's at odds with the docs, but aside from that I don't see any particular problem with passing in the whole object here. Thanks for looking into it!